_config.yml

```diff
@@ -83,6 +83,14 @@ theme: aframe
 exclude_generator:
 - index
 
+# Feed
+## Plugin https://github.com/hexojs/hexo-generator-feed
+feed:
+  type: atom
+  path: feed.xml
```

Comment:
can you rename this to `atom.xml` or `feeds/atom.xml`? just in case we decide to switch to RSS2 at some point